src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/CommentsIndentation/context/case3/Test.java

```diff
@@ -0,0 +1,22 @@
+package TreeWalker.CommentsIndentation;
+
+public class Test {
```

Comment:
add another method with a suppressed violation to this class to show that filter doesn't increase its scope to a CLASS_DEF.

Reply:
done